blog-details.html

```diff
@@ -0,0 +1,610 @@
+          <li class="navigation__list-item">
+            <a
+              href="#"
+              class="navigation__link button button__small button__navigation"
+            >
+              About
+            </a>
+          </li>
+          <li class="navigation__list-item">
+            <a
+              href="./catalog.html"
+              class="navigation__link button button__small button__navigation"
+            >
+              Products
+            </a>
+          </li>
+          <li class="navigation__list-item">
+            <a
+              href="#"
+              class="navigation__link button button__small button__navigation"
+            >
+              Creators
+            </a>
+          </li>
+          <li class="navigation__list-item">
+            <a
+              href="./blog-details.html"
+              class="navigation__link button button__small button__navigation"
+            >
+              Blog
+            </a>
+          </li>
+          <li class="navigation__list-item">
+            <a
+              href="./contacts.html"
+              class="navigation__link button button__small button__navigation"
+            >
+              Contact
+            </a>
+          </li>
+        </ul>
+      </nav>
+      <div class="header__assets-block list">
+        <form action="#" method="get" class="assets-block__searching">
+          <label for="asset-searching" class="searching__label hidden"
+            >Find asset</label
+          >
+          <input
+            type="text"
+            class="searching__input input input__small"
+            id="asset-searching"
+            placeholder="Search for Digital Assets"
+          />
+          <button type="submit" class="searching__button hidden">Search</button>
+        </form>
+        <a
+          href="#"
+          target="_blank"
+          class="assets-block__button button button__primary button__square-medium"
+        >
+          <svg class="assets-block__icon icon">
+            <use xlink:href="./images/icons/sprite.svg#bag"></use>
+          </svg>
+        </a>
+      </div>
+    </header>
+    <main class="container__main">
+      <section class="blog-introduction container__section list list__column">
+        <span class="blog-introduction__tag label"> Typography</span>
+        <h1
+          class="blog-introduction__title heading__main-title main-information__highlite-text"
+        >
+          Mastering the Art of Typography: Tips&nbsp;for Eye-Catching Designs
+          for 2024
+        </h1>
+      </section>
+      <section class="blog container__section">
+        <div class="blog__image-container">
+          <img
+            src="./images/intros/bisness-presentation.webp"
+            alt="best bisness plan presentation templates"
+            width="1280"
+            height="414"
+            class="blog__image"
+          />
+        </div>
+        <aside class="blog__article-information list list__column">
```

Comment:
This section has important information about author and this post, so it cannot be Aside tag.

Reply:
Fixed 🥇